src/api.js

```diff
@@ -2,7 +2,8 @@ import fetch from 'cross-fetch';
 import './style.css';
 
 const key = '4367d242d87843ddb5e0a8cc46a359d5';
-const url = `https://api.rawg.io/api/games?key=${key}`;
+const quantity = 50;
+const url = `https://api.rawg.io/api/games?key=${key}&page_size=${quantity}`;
 // const gameUrl = `https://api.rawg.io/api/games/${id}`;
```

Comment:
To make your code clean, It's a good practice to get rid of used commented codes.